src/logging.cpp

```diff
@@ -165,6 +164,8 @@ const CLogCategoryDesc LogCategories[] =
 #endif
     {BCLog::UTIL, "util"},
     {BCLog::BLOCKSTORE, "blockstorage"},
+    {BCLog::UNCONDITIONAL_RATE_LIMITED, "uncond_rate_limited"},
+    {BCLog::UNCONDITIONAL_ALWAYS, "uncond_always"},
```

Comment:
Just to clarify - all the other categories (`QT`, `REINDEX`, `BENCH`, etc.) are not rate-limited?

Reply:
Yes, only uses of `LogPrintf` (`LogPrint(BCLog::UNCONDITIONAL_RATE_LIMITED, ...)`) are rate limited.